apiary-metastore-events/README.md

```diff
@@ -5,6 +5,50 @@ Apiary Metastore Events contains a set of modules responsible for retrieving and
 
 Currently, the following modules are defined:
  - [sns-metastore-events](sns-metastore-events) - SNS Metastore Events contains a set of modules responsible for retrieving and processing Hive Metastore events for SNS.
+ - [kafka-metastore-events](kafka-metastore-events) - Kafka Metastore Events contains a set of modules responsible for retrieving and processing Hive Metastore events for Kafka.
+
+## Kafka Configuration
+
+To configure the receiver or listener, use the configuration below and prefix all parameters with `com.expediagroup.apiary.extensions.events.metastore.kafka.messaging.`.
```

Comment:
It would be good to mention here that these configurations need to be put in the `hive-site.xml` and perhaps provide an example:

<property>
  <name>com.expediagroup.apiary.extensions.events.metastore.kafka.messaging.bootstrap.servers</name>
  <value>KAFKA_BOOTSTRAP_SERVERS</value>
</property>

This PR is merged. So, perhaps, these can be included with the changes you were planning to make to the receiver.